### PR TITLE
Enabled collaboration behaviour in the automation section

### PR DESCRIPTION
**Pull request by deanhannigan**
## Description

Collaboration has been extended to the automation section. 

- See what everyone is currently working on in the automation section!
- Live broadcast of any changes made by collaborators working in your application automations.

Addresses: 
- BUDI-7151

## Screenshots

Screenshot 2023-07-17 at 11 36 22

### CODE REVIEW

File: packages/builder/src/builderStore/store/automation/index.js [hunk 1]
```diff
@@ -248,4 +248,39 @@ const automationActions = store => ({
...
+    }
+    if (!automation) {
+      store.update(state => {
+        // Remove the automation
+        state.automations = state.automations.filter(
+          x => x._id !== automationId
+        )
+        // Select a new automation if required
+        if (automationId === state.selectedAutomationId) {
+          store.actions.select(state.automations[0]?._id)
```
Comment (aptkingston) [inline packages/builder/src/builderStore/store/automation/index.js:263]: NAB: won't do anything bad having this here I don't think, but is this needed? I would have expected that the URL state sync util would correctly recognise that the selected automation ID was no longer valid and trigger a redirect to select a new one.
That's what happens with other entities (screens etc).

**Review by aptkingston — APPROVED** [review]
LGTM! Will have a play locally too but code looks great.

File: packages/builder/src/builderStore/store/automation/index.js (hunk 1 above)
Comment (deanhannigan) [reply packages/builder/src/builderStore/store/automation/index.js:263]: Yep, I did a bit a re-testing there and the state sync redirect stops it from becoming an issue  :thumbsup:

Comment (aptkingston) [reply packages/builder/src/builderStore/store/automation/index.js:260]: Cool, lets just remove that then and we can merge after.

**Pull request merged**